_rules/disable-shortcut-670a30.md

```diff
@@ -0,0 +1,273 @@
+---
+id: 670a30
+name: Printable keys only shortcut can be disabled
+rule_type: atomic
+description: |
+  This rule checks that if keyboard shortcuts are implemented using only printable characters, a mechanism to disable the shortcut exists.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys.
+
+## Expectation
+
+For each [user interface component][] that is a [descendent][] of the root node of the test target, if a [printable character][] [shortcut][keyboard shortcut] exists to trigger the action of that [user interface component][], it can be disabled.
```

Comment:
If we want to restrict disabling to UI component, we should mention it in the first part of the expectation.
```suggestion
For each [user interface component][] that is a [descendent][] of the root node of the test target, if a [printable character][] [shortcut][keyboard shortcut] exists to trigger the action of that [user interface component][], it can be disabled by another [user interface component][].
```
(this applies to other rules with similar expectation)